amespahdbpythonsuite/transitions.py

```diff
@@ -49,10 +49,10 @@ def __set(self, d=None, **keywords):
         if d:
             if d.get('type', '') == self.__class__.__name__:
                 if not keywords.get('shift'):
-                    self.__shift = d['shift']
+                    self._shift = d['shift']
```

Comment:
Our underscores are not really consistent yet. My understanding is that one uses single underscores for variables and functions that should not be accessed outside the class, but possibly from derived classes, and double underscores for variables and functions that cannot be accessed outside the class in any way. Many class attributed could be ported to the single underscore in our code I think ...

Reply:
Single underscore is used to hint the programmer this variable/object should be used internally in the class, however these objects can still be accessed from another script/class. The double underscore essentially changes the variables attribute name and adds the class name in order to avoid naming collisions in subclasses or the attribute getting overridden. Everything can still be accessed on way or another.